### PR TITLE
Rename user trigger fields for consistency

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: priv/repo/migrations/20190327104558_migrate_repeating_trigger_embedded_field.exs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+defmodule Sanbase.Repo.Migrations.MigrateRepeatingTriggerEmbeddedField do
+  use Ecto.Migration
+
+  def up do
+    Application.ensure_all_started(:timex)
```
Comment (tspenov) [inline priv/repo/migrations/20190327104558_migrate_repeating_trigger_embedded_field.exs]: why do we need this one ?

File: mix.exs [hunk 2]
```diff
@@ -100,7 +100,8 @@ defmodule Sanbase.Mixfile do
       {:number, "~> 1.0"},
       {:remote_ip, "~> 0.1"},
       {:vex, "~> 0.8.0", override: true},
-      {:stream_data, "~> 0.4.2", only: :test}
+      {:stream_data, "~> 0.4.2", only: :test},
+      {:tzdata, "~> 0.5"}
```
Comment (tspenov) [inline mix.exs]: why `:tzdata` is added ?

File: priv/repo/migrations/20190327104558_migrate_repeating_trigger_embedded_field.exs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    execute("""
+      UPDATE user_triggers
+      SET trigger = trigger - 'repeating' || jsonb_build_object('is_repeating', trigger->'repeating')
```
Comment (tspenov) [inline priv/repo/migrations/20190327104558_migrate_repeating_trigger_embedded_field.exs]: Did you find a way to update second level key/value ? like something is `settings`

File: mix.exs (hunk 2 above)
Comment (IvanIvanoff) [reply mix.exs]: timex depends on it so we implicitly have it, but without adding it here the tests failed with some unexpected (for me) errors. You can check the error builds here: https://jenkins.internal.santiment.net/job/Santiment/job/sanbase2/job/PR-1076/

File: priv/repo/migrations/20190327104558_migrate_repeating_trigger_embedded_field.exs (hunk 1 above)
Comment (IvanIvanoff) [reply priv/repo/migrations/20190327104558_migrate_repeating_trigger_embedded_field.exs]: I had some issues as linked in the other comment

**Pull request merged**